src/_guides/language/language-tour.md

```diff
@@ -4241,6 +4240,33 @@ Keep in mind the following when you use deferred loading:
   using <code>deferred as <em>namespace</em></code>.
   The `loadLibrary()` function returns a [`Future`](/guides/libraries/library-tour#future).
 
+#### The `library` directive
+
+The only reason to use a library directive is
+to attach library-level [doc comments][] or [metadata annotations][].
```

Comment:
**issue:** This can be simplified. Consider making this more concise.

Reply:
I really can't imagine a better way to say this. It's a pretty simple sentence that is exactly the point that needs to be conveyed.